controllers/patientsController.js

```diff
@@ -230,3 +230,28 @@ module.exports.createNewPatient = async (req, res) => {
     return res.redirect('/psychologue/nouveau-patient');
   }
 };
+
+module.exports.deletePatientValidators = [
+  check('patientId')
+    .isUUID()
+    .withMessage('Vous devez spécifier un patient à supprimer.'),
+];
+
+module.exports.deletePatient = async (req, res) => {
+  if (!validation.checkErrors(req)) {
+    return res.redirect('/psychologue/mes-seances');
+  }
+
+  const { patientId } = req.body;
+  try {
+    const psychologistId = cookie.getCurrentPsyId(req);
+    await dbPatients.deletePatient(patientId, psychologistId);
+    console.log(`Patient deleted ${patientId} by psy id ${psychologistId}`);
```

Comment:
We're gonna have the console log twice, could be confusing

Reply:
Indeed, it was done like this for the appointments so I kept it. What do you think @polomarcus should we keep only one log in both places (appointment and patient deletion) ?

Reply:
If it is the patient that is deleted, we should keep only the patient's message so that it is clearer.

Reply:
I'm not sure to understand.
Today, when we delete a patient, we have 2 logs :
```
Patient id a8d06dad-76be-49b8-8c4e-337d61b8a4b7 deleted by psy id 9a42d12f-8328-4545-8da3-11250f876146
Patient deleted a8d06dad-76be-49b8-8c4e-337d61b8a4b7 by psy id 9a42d12f-8328-4545-8da3-11250f876146
```
The first one comes from the controller, the second one from the db.
Should we keep it like this or remove one ?
If we remove one, I would keep the controller one and delete the db one.

Reply:
I'm ok with keeping both (especially for the error) if you prefix them by CONTROLLER and DB :)

Reply:
We'll do it in another PR to handle all logs the same way